egs/chime4/ConvTasNet/local/create_metadata.py

```diff
@@ -0,0 +1,106 @@
+import os
+import argparse
+from glob import glob
+import pandas as pd
+import numpy as np
+
+# Command line arguments
+parser = argparse.ArgumentParser()
+parser.add_argument("--chime3_dir", type=str, default=None, help="Path to CHiME3 root directory")
+
+# Set seed for random generation
+SEED = 4
+np.random.seed(SEED)
+
+
+def main(args):
+    chime3_dir = args.chime3_dir
+    create_local_metadata(chime3_dir)
+
+
+def create_local_metadata(chime3_dir):
+    # Get CHiME-3 annotation files
+    c3_annot_files = [
+        f for f in glob(os.path.join(chime3_dir, "data", "annotations", "*real*.json"))
+    ]
+    # Get CHiME-4 annotation files
+    c4_annot_files = [
+        f for f in glob(os.path.join(chime3_dir, "data", "annotations", "*real*.list"))
+    ]
+    for c3_annot_file_path in c3_annot_files:
+        # Read CHiME-3 annotation file
```

Comment:
I think it could be helpful to move the entire `for` loop body into a separate function so as to make it obvious that all loop iterations are entirely independent.